### DISCUSSION TITLE
[Feature] Album Groups

### DISCUSSION THREAD

**Discussion by hv-dev** [immich-app/immich#12805] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Grouping Albums will provide greater organisation of libraries for both personal and professional use.
Example:

Folder / Group: Event Name
  Subfolders:
    - Phone Camera
    - DSLR Camera
    - Friends Photos

This feature would also bring improvement to the folder structure for large libraries

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Thanks for the request. duplicate of #2073
